src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -2499,6 +2499,37 @@ private static Stream GetStreamFromString(string s)
         stream.Position = 0;
         return stream;
     }
+
+    [ActiveIssue(12799)]
```

Comment:
Can we split this test so the ctor of SoapAttributes etc. can be covered in a test that is not skipped?